src/macaron/code_analyzer/call_graph.py

```diff
@@ -7,11 +7,14 @@
 from typing import Generic, TypeVar
 
 Node = TypeVar("Node", bound="BaseNode")
-"""This binds type ``Node`` to ``BaseNode`` and any of its subclasses.
-
-Therefore, any node of type ``Node`` that is stored in the call graph
-container will be a subtype of ``BaseNode``.
-"""
+# This code documentation is commented out because newer Sphinx version (^=6.1.0) has issues processing
```

Comment:
Have you tried adding `sphinx_toolbox.more_autodoc.typevars` to `conf.py`?
See https://sphinx-toolbox.readthedocs.io/en/latest/extensions/more_autodoc/typevars.html

Reply:
I have tried but I still received the same error.